README.md

```diff
@@ -4,17 +4,24 @@ riscv-tests
 About
 -----------
 
-This repository hosts unit tests for RISC-V processors.
+This repository hosts unit tests for RISC-V processors including the instructions of the Xpulpimg extension subsets as defined [here](https://iis-people.ee.ethz.ch/~balasr/pulp-ext/pulp.html).
```

Comment:
Sorry for being pedantic :) We decided that Xpulpimg should not be in this anymore, right? Probably you meant Xpulp instead of Xpulpimg, also becasue the link "[here](https://iis-people.ee.ethz.ch/~balasr/pulp-ext/pulp.html)" doesn't contain anything about Xpulpimg.